x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/mustache_templates.ts

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
this is a copy of the `spaces_only` tests, but only included a test for the new action variable, since it's now different between the two